ApiDoctor.Publishing/CSDL/csdlwriter.cs

```diff
@@ -1589,19 +1594,39 @@ private T ConvertParameterToProperty<T>(string typeName, ParameterDefinition par
                 if (descriptionTerm != null)
                 {
                     LogIfDifferent(descriptionTerm.String, sourceParameter.Description, issues, $"Type {typeName} has a different value for term '{termForDescription}' than the documentation.");
+                    if (string.IsNullOrWhiteSpace(descriptionTerm.String))
+                    {
+                        descriptionTerm.String = sourceParameter.Description;
+                    }
                 }
                 else
                 {
                     targetProperty.Annotation.Add(
                     new Annotation()
                     {
                         Term = termForDescription,
-                        String = sourceParameter.Description.ToStringClean(),
+                        String = sourceParameter.Description,
                     });
                 }
             }
         }
 
+        private string RemoveUnnecessaryInformationFromDescriptionAnnotation(string description)
+        {
+            var termsToRemove = new string[] {
```

Comment:
We could probably move this collection to be a private read-only variable of the class to avoid its potential allocation every time the function is called since it doesn't change.
Also, we should probably change it to be HashSet rather than array and initialize it as below in the event we want to ignore the case.

```cs
new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```